run/README.namelist

```diff
@@ -504,22 +508,32 @@ Namelist variables for controlling the adaptive time step option:
- qna_update                          = 0   ! set to 1 to update time-varying sfc aerosol emission from climatology or real-time data 
+ qna_update                          = 0   ! set to 1 to update time-varying sfc aerosol emission from climatology or 
+                                             real-time data 
                                              with mp_physics = 28. Use with input file ‘wrfqnainp_d0*’
                                              (must set auxinput17_interval and io_form_auxinput17; new in 4.4)
- wif_fire_emit                       = .false.   ! set to .true. to include biomass burning organic and black carbon aerosols with mp_physics = 28 (new in 4.4)
- wif_fire_inj                        = 1   ! (default) vertically distribute biomass burning emissions in mp_physics = 28 (new in 4.4)
+ wif_fire_emit                       = .false.   ! set to .true. to include biomass burning organic and black carbon 
+                                             aerosols with mp_physics = 28 (new in 4.4)
+ wif_fire_inj                        = 1   ! (default) vertically distribute biomass burning emissions 
+                                             in mp_physics = 28 (new in 4.4)
                                      = 30, HUJI (Hebrew University of Jerusalem, Israel) spectral bin microphysics,
                                            fast version
                                      = 32, HUJI spectral bin microphysics, full version
-                                     = 40, Morrison (2 moments) with consideration of CESM-NCSU RCP4.5 climatological aerosol 
+                                     = 38, Thompson scheme with 2-moment graupel/hail (new in 4.5)
+                                     = 40, Morrison (2 moments) with consideration of CESM-NCSU RCP4.5 climatological 
+                                           aerosol 
                                      = 50, P3 1-ice category, 1-moment cloud water
                                      = 51, P3 1-ice category plus double-moment cloud water
                                      = 52, P3 2-ice categories plus double-moment cloud water
                                      = 53, P3 1-ice category, 3-moment ice, plus double-moment cloud water
-                                     = 55, Jensen-ISHMAEL (Ice-Spheroids Habit Model with Aspect-ratio Evolution) scheme                                            predicting qc, qr, and three ice species with four predictive equations 
-                                           for each ice
+                                     = 55, Jensen-ISHMAEL (Ice-Spheroids Habit Model with Aspect-ratio Evolution) scheme 
+                                           predicting qc, qr, and three ice species with four predictive equations for each ice
                                      = 56, NTU multi-moment scheme  ! for ntu3m
+ ccnty                                 = 1  ! for aerosol background type, marine, for ntu3m
```

Comment:
This option and description already exists further down in the file (line 561).

Reply:
@kkeene44 Thanks for the file. I will select some to add. If you see any that may appear important to you or users, let me know.

Reply:
@kkeene44 I moved ccnty from below to the current location near the option it is supposed to be associated with.